buildSrc/src/main/groovy/io.micronaut.build.internal.testing-base.gradle

```diff
@@ -5,14 +5,13 @@ plugins {
 
 group "io.micronaut.example"
 
-repositories {
-    mavenCentral()
-}
-
 sourceCompatibility = 1.8
 targetCompatibility = 1.8
 
-configurations {
-    all*.exclude module: "byte-buddy"
-    all*.exclude module: "javassist"
-}
+tasks.named("test") {
+    def testExamples = project.rootProject.tasks.findByName("testExamples")
+    if (!testExamples) {
+        testExamples = project.rootProject.tasks.create("testExamples")
+    }
+    testExamples.dependsOn(it)
```

Comment:
probably shouldn't do this /cc  @melix

Reply:
It's indeed a suspicious pattern :) What are you trying to do and what is creating the `testExamples` task?

Reply:
I'm just trying to run all examples tests with one task

Reply:
Oh I see. Then you simply need to do:

```
tasks.register("testExamples") {
    dependsOn "test"
}
```